tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -40,9 +41,14 @@ var WITView = widgets.DOMWidgetView.extend({
     const height = parseInt(
       this.model.attributes.layout.attributes.height, 10) - 20;
     const iframe = document.createElement('iframe');
-    const templateLocation =
-      window.__webpack_public_path__ + 'wit_jupyter.html';
-    const src = `<link rel="import" href="${templateLocation}">
+
+    // Adjust WIT html location if running in a jupyter notebook
+    // and not in jupyterlab.
+    if (document.querySelector('body').getAttribute('data-base-url') != null) {
```

Comment:
do you know what the value of data-base-url would be? would that make it a more robust check? I admit this is a hard thing to test. Is there a version string somewhere that might have the project name or something.

Reply:
I believe it may depend on your own Jupyter installation, so I'm not looking to test a specific value of that field, just that it exists